ci/Jenkinsfile.e2e-nightly

```
@@ -0,0 +1,50 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@v1.7.9'
+
+pipeline {
+  agent { label 'linux || macos || windows' }
+
+  parameters {
+    string(
+      name: 'BRANCH',
+      description: 'Name of the branch to checkout and build.',
+      defaultValue: 'develop',
+    )
+  }
+
+  options {
+    timestamps()
+    disableConcurrentBuilds()
+      /* Prevent Jenkins jobs from running forever */
+      //timeout(time: 40, unit: 'MINUTES')
+      /* Allow copying of artifacts from this job. */
+      //copyArtifactPermission('/status-mobile/e2e/*')
+      /* Limit builds retained */
+      //buildDiscarder(logRotator(
+      //      numToKeepStr: '10',
+      //      daysToKeepStr: '30',
+      //      artifactNumToKeepStr: '10',
+      //      ))
+  }
+
+  stages {
+    stage('Build') {
+      steps {
+        script {
+          apk_build = jenkins.Build('status-mobile/platforms/android-e2e')
+          apk_build_number = apk_build.getNumber().toString()
+        }
+      }
+    }
+    stage('Run e2e') {
+      steps {
+        build job: 'status-mobile/e2e/status-app-nightly',
+              parameters: [
+                string(name: 'APK_BUILD_NUMBER', value: apk_build_number),
+                string(name: 'BRANCH', value: env.BRANCH),
+              ]
```

Use `build` like a function, with parenthesis: `build(...)`
```suggestion
        build(
          job: 'status-mobile/e2e/status-app-nightly',
          parameters: [
            string(name: 'APK_BUILD_NUMBER', value: apk_build_number),
            string(name: 'BRANCH', value: env.BRANCH),
          ]
        )
```